components/image/image.css

```diff
@@ -0,0 +1,23 @@
+.img-responsive-container{
+    max-width: 800px;
+}
+.img-container{
+    max-width: 300px;
+}
+.img-responsive{
+    width: 100%;
+    object-position: top center;  
+    object-fit: cover; 
+}
+.img-semi-square{
+    width: 100%;
+    border-radius: 10%;
+    object-position: top center;  
+    object-fit: cover; 
```

Comment:
You can make ageneric class with any name and give those properties to that class and add it to evry image you add , , one more suggestion try to add images of different sizes too and provide classes for those to user

Reply:
Done the first one but for the second one we can have any size of parent container and the images will cover the parent container so there is no use of giving different classes for sizes.